index.md

```diff
@@ -23,6 +25,63 @@ web-based interface for your servers
 [![Docker screenshot]({{ site.baseurl }}/images/site/screenshot-docker.png)]({{ site.baseurl }}/images/site/screenshot-docker.png){:.screenshot.zoom}
 {% endcapture %}
 
+{% capture intro-text %}
+## Introducing Cockpit
+
+Cockpit is a web-based graphical interface for servers and is intended for everyone, especially those who are:
+
+{:.audience-list}
+- **new to Linux** (including Windows admins)
+- **familiar with Linux** and want an easy, graphical way to administer servers
+- **expert admins** who use other tools and want to quickly check-in on systems
+
+## Cockpit is (nearly) everywhere
+
+You can install Cockpit on the major distributions, including:
+
+{:.distro-logos}
+{%
+  for distro in site.data.distros
+%}- [![{{ distro.name }}](/images/site/os-{{ distro.first}}.svg)]({{ site.baseurl }}/running.html#{{ distro.first }})
+{% endfor %}
+
+Once Cockpit is up and running, you can access systems from all major web browsers on operating systems (and, yes, this even includes Windows and macOS).
```

Comment:
```suggestion
Once Cockpit is up and running, you can access systems from all major web browsers on any operating system (including Windows, MacOS, and Android).
```

I forget, which one was the problematic one with the self-signed certs? iOS or MacOS?

Reply:
iOS is the problem. It's due to certs and only having one web browser, Safari (the others are just reskinned UIs around Safari's WebKit), so any isn't quite right. I had "all" earlier but removed it.

All major web browsers → Safari does actually work on macOS, so it's not a problem due to WebKit (or even Safari's version of it), but the differing implementation on iOS.